elasticsearch-agent/src/main/resources/templates/data_storage.json

```diff
@@ -5,25 +5,25 @@
         "doc_type": { "type": "keyword", "store": true },
         "id": {"type": "keyword", "store": true},
         "parentId": {"type": "keyword", "store": true, "index": false },
-        "name": {"type": "keyword", "store": true},
+        "name": {"type": "text", "store": true},
         "path": { "type": "text", "analyzer": "file_path_analyzer"},
         "createdDate": { "type": "date", "format": "yyyy-MM-dd HH:mm:ss.SSS" },
         "description": { "type": "text", "store": true },
-        "storageType": { "type": "keyword" },
+        "storageType": { "type": "text" },
         "regionCode": { "type": "keyword" },
         "ownerUserId": { "type": "long" },
-        "ownerUserName": { "type": "keyword" },
```

Comment:
Please, do not change type for all security related fields: `owner..., allowed..., denied...`